pkg/distributor/distributor.go

```diff
@@ -261,8 +261,9 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Help: "The total number of received requests, excluding rejected and deduped requests.",
 		}, []string{"user"}),
 		receivedSamples: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
-			Name: "cortex_distributor_received_samples_total",
-			Help: "The total number of received samples, excluding rejected and deduped samples.",
+			Name:        "cortex_distributor_received_samples_total",
+			Help:        "The total number of received samples, excluding rejected and deduped samples.",
+			ConstLabels: prometheus.Labels{"destination": "ingester"},
```

Comment:
I'm not sure the label makes sense here for "received" samples. If it was forwarded or sent metric, then yes.

By introducing `cortex_distributor_received_samples_total` with different destinations, many panels and recording rules will need to change to be adjusted to preserve their existing meaning.